README.md

```diff
@@ -26,10 +26,18 @@ See ``config.yaml.dist`` for a configuration example.
 
 ## Run
 
+Native:
+
 ```
 $ ./mqtt_blackbox_exporter -config.file config.yaml
 ```
 
+Using Docker:
+
+```
+docker run --rm -it -p 9142:9142 -v ${PWD}/:/data/ inovex/mqtt_blackbox_exporter:latest -config.file /data/config.yaml
```

Comment:
How about reserving a dedicated port (see https://prometheus.io/docs/instrumenting/writing_exporters/#port-numbers) and list this exporter at https://github.com/prometheus/prometheus/wiki/Default-port-allocations.

Also port 9142 is already used for the BigIP exporter (https://github.com/ExpressenAB/bigip_exporter)